.github/workflows/linter.yml

```diff
@@ -21,4 +27,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.60.1
+          version: v1.60.1
```

Comment:
nit: New line

Reply:
I am confused. Both of these the same version

Reply:
I believe a new line is missing at the end of files, might've been formatted via IDE. Just a nit.

Reply:
Added a new line at EOF